pkgs/applications/misc/nut/default.nix

```diff
@@ -63,16 +71,23 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     substituteInPlace $out/lib/systemd/system-shutdown/nutshutdown \
+      --replace /bin/sed "${gnused}/bin/sed" \
       --replace /bin/sleep "${coreutils}/bin/sleep" \
       --replace /bin/systemctl "${systemd}/bin/systemctl"
 
     for file in system/{nut-monitor.service,nut-driver-enumerator.service,nut-server.service,nut-driver@.service} system-shutdown/nutshutdown; do
-    substituteInPlace $out/lib/systemd/$file \
-      --replace "$out/etc/nut.conf" "/etc/nut.conf"
+      substituteInPlace $out/lib/systemd/$file \
+        --replace "$out/etc/nut.conf" "/etc/nut/nut.conf"
```

Comment:
please fix indentation

Reply:
Could you expand on what is wrong with the indentation?

Previously the body of the `for` loop was not indented (as pointed out in earlier feedback) which I corrected

Reply:
Sorry @pbsds, [I asked him to do this](https://github.com/NixOS/nixpkgs/pull/273397#discussion_r1423237301) since I noted that it wasn't indented yet was in a `for` loop.

Reply:
oh, I'm blind, sorry. The for expression so long it wraps which threw me off.